docs/releasehistory.rst

```diff
@@ -1,12 +1,17 @@
 Release History
 ***************
 
-Current development
-===================
+0.18.3 - Disk writing enhancements and bugfixes
```

Comment:
Maybe "Storage enhancements" instead of "Disk writing enhancements"?

Reply:
Yes, that's better, thank you!